app/controllers/requests_controller.rb

```diff
@@ -23,8 +23,9 @@ def edit
   # POST /requests
   def create
     @request = Request.new(request_params)
-
     if @request.save
+      Mailer.send_generic_email(false, @request.email, 'workshop.portal@gmail.com', I18n.t('requests.email.topic'),
```

Comment:
I think you should add a test for this line. Not for testing the mailer functionality, but rather for testing that a mail is sent. Otherwise one could delete this line, and the tests would be green, allthough no email was sent.
Maybe something like 
```
it "sends an email" do
  expect{ post :create, request: valid_attributes, session: valid_session }.to change{ActionMailer::Base.deliveries.count}.by(1)
end
```